.github/workflows/ci.yml

```diff
@@ -5,43 +5,48 @@ on:
   pull_request:
   push:
     branches:
```

Comment:
*arbitrary comment location*

It might be good to add "Alpha Test" or something in the top left next to the logo to emphasize the early nature.

Reply:
Updated the logo in d0ecf47, lmk how that looks